src/main/groovy/org/octopusden/octopus/license/management/plugins/gradle/LicenseGradlePlugin.groovy

```diff
@@ -37,12 +38,13 @@ class LicenseGradlePlugin implements Plugin<Project> {
         }
     }
 
-    private boolean isTrue(Project project, String property) {
-        return 'true'.equalsIgnoreCase(project.rootProject.findProperty(property) as String ?: 'true')
+    private boolean isFalse(Project project, String property) {
```

Comment:
The reason I changed the `isTrue` logic is because the function will be only used in the `nodeOnlyIf` function, which then will invert the value with `!`. The isTrue function is meant specifically for checking if certain parameters, like `license.skip` and `node.skip`, have the value `true` and then invert it to `false`.

If I kept the original isTrue logic, it would require the use of an `AND` operator to achieve the same behavior, which would make the logic harder to follow. In my view, using `AND` in this case isn’t as intuitive or aligned with the intended purpose of checking whether a particular configuration parameter should evaluate to `true` or `false`. By adjusting the function to reflect the negative case directly, it becomes clearer and more aligned with the intended checks.